cumulus/primitives/utility/src/lib.rs

```diff
@@ -264,7 +263,7 @@ impl<
 }
 
 /// XCM fee depositor to which we implement the TakeRevenue trait
-/// It receives a Transact implemented argument, a 32 byte convertible acocuntId, and the fee
+/// It receives a Transact implemented argument, a 32 byte convertible accountId, and the fee
```

Comment:
```suggestion
/// It receives a `Transact` implemented argument and a 32 byte convertible `AccountId`, and the fee
```

Reply:
Can't suggest on multiple lines here but the whole thing should be:
```rust
/// XCM fee depositor to which we implement the `TakeRevenue` trait.
/// It receives a `Transact` implemented argument and a 32 byte convertible `AccountId`, and the fee
/// receiver account's `FungiblesMutateAdapter` should be identical to that implemented by `WithdrawAsset`.
```

Reply:
Fixed